addons/crm/crm_view.xml

```diff
@@ -79,19 +79,19 @@
                             <field name="name"/>
                             <field name="sequence"/>
                             <field name="type"/>
-                            <label for="probability" />
-                            <div>
+                            <field name="on_change"/>
+                            <label for="probability" attrs="{'invisible': [('on_change', '=', False)]}"/>
+                            <div attrs="{'invisible': [('on_change', '=', False)]}">
                                 <field name="probability"/>
                                 <p class="text-muted">
-                                    A stage will be considered as 'Won' if the probability is 100% and it is folded in Kanban View.<br/>
-                                    A stage will be considered as  'Lost (Dead)' if the probability is 0% and it is folded in Kanban View.
+                                    A stage will be considered as 'Won' if the probability is 100% and Check box 'Change Probability Automatically' is true and sequence is greater than 1.<br/>
```

Comment:
A stage will be considered as 'Won' if the probability is 100% and if the 'Change Probability Automatically' is true. The sequenc eshould be greather than 1.